src/Mod/Path/libarea/Adaptive.cpp

```diff
@@ -146,6 +146,38 @@ inline DoublePoint GetPathDirectionV(const Path &pth, size_t pointIndex)
 	const IntPoint &p2 = pth.at(pointIndex);
 	return DirectionV(p1, p2);
 }
+	
+bool doFilter(Path &ppg) {
```

Comment:
I suggest using a different variable name, since this one below refers to a vector of paths, rather than a single path.

Reply:
ok